move/sources/canvas/token.move

```diff
@@ -64,6 +61,12 @@ module addr::canvas_token {
     /// Vectors provided to draw were of different lengths.
     const E_INVALID_VECTOR_LENGTHS: u64 = 9;
 
+    /// The caller tried to call a function that requires collection owner privileges.
+    const E_CALLER_NOT_COLLECTION_OWNER: u64 = 10;
+
+    /// The caller exceeds the max number of pixels per draw.
+    const E_EXCEED_MAX_NUMBER_OF_PIXELS_PER_DRAW: u64 = 11;
+
```

Comment:
can we make the error messages more user friendly? They show up in the txn result

Reply:
can you give me a friendly version? sorry im terrible at writing